### PR TITLE
feature(CI): build and push edxapp now creates releases

### PR DESCRIPTION
**Pull request by michaelroudnitski**

### CODE REVIEW

**Review by danielrudn — APPROVED** [review]

**Review by ExiaSR — COMMENTED** [review]
I am not sure if this is the right approach.

Usually, we tag the commit or create a new release, then CI picks up the event and builds an image with the incoming tag.

Moreover, we likely will be using this repository to build more images, like forum, notes API server, etc, so the tag name should be namespaced

such as `edxapp-0.3.0`, `forum-0.3.0`

I was going to implement something like this in this new workflow, but didn't have the time to implement the tag name parsing https://github.com/ibm-skills-network/configuration/blob/open-release/juniper.master/.github/workflows/push.yml

**Comment by michaelroudnitski** [comment]
> I am not sure if this is the right approach.
> 
> Usually, we tag the commit or create a new release, then CI picks up the event and builds an image with the incoming tag.
> 
> Moreover, we likely will be using this repository to build more images, like forum, notes API server, etc, so the tag name should be namespaced
> 
> such as `edxapp-0.3.0`, `forum-0.3.0`
> 
> I was going to implement something like this in this new workflow, but didn't have the time to implement the tag name parsing https://github.com/ibm-skills-network/configuration/blob/open-release/juniper.master/.github/workflows/push.yml

Good points, it didn't take long to do so if it will be helpful for now let's merge and replace it later, otherwise we can close. Up to you

**Comment by ExiaSR** [comment]
Let's merge it

File: .github/workflows/push-edxapp.yml [hunk 1]
```diff
@@ -37,5 +37,15 @@ jobs:
...
       - name: Push
         run: docker push us.icr.io/skills-network-portals/edxapp:${{ github.event.inputs.tag }}
+      - name: Create Release
+        uses: actions/create-release@v1
+        id: create_release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.event.inputs.tag }}
+          release_name: ${{ github.event.inputs.tag }}
```
Comment (ExiaSR) [inline .github/workflows/push-edxapp.yml:47]:
```suggestion
          tag_name: "edxapp-${{ github.event.inputs.tag }}"
          release_name: "edxapp-${{ github.event.inputs.tag }}"
```

**Pull request merged**